### ISSUE TITLE
Connecting to deprecated signal QDBusConnectionInterface::serviceOwnerChanged

### ISSUE THREAD

**Issue by shakaran** [jlherren/skype-call-recorder#9]
Using in Ubuntu 16.10:

Git commit: f9fc081de3f7812cef39009574df48ec0208a671
Build date: 2016-03-25

I get this warning:

    Connecting to deprecated signal QDBusConnectionInterface::serviceOwnerChanged(QString,QString,QString)

**Comment by jlherren** [comment]
As far as I know this warning has been around for quite a while. Are you having any issues other than the warning?

**Comment by shakaran** [comment]
@jlherren no, it seems work perfectly, but probably in new QT version could be a problem.

It seems that the code is available from QT 4.2, working fine in [4.6](http://doc.qt.io/qt-4.6/qdbusconnectioninterface.html#serviceOwnerChanged), but [in 5.6](http://doc.qt.io/qt-5.6/qdbusconnectioninterface.html#serviceOwnerChanged) and [5.7](http://doc.qt.io/qt-5.7/qdbusconnectioninterface.html#serviceOwnerChanged) seems deprecated or removed.
